bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/NIFS_BP.xml

```diff
@@ -126,7 +129,7 @@ SCIENCE DARKS:
       <container kind="observation" type="Observation" version="2014A-1" subtype="basic" key="dfe11324-c32c-4dac-a70a-06e87d8db3f6" name="NIFS-BP-1">
         <paramset name="Observation" kind="dataObj">
           <param name="title" value="Acq: Before Standard"/>
-          <param name="libraryId" value="0"/>
+          <param name="libraryId" value="1"/>
```

Comment:
These are referenced in the code, usually.  If the ID changes there should be corresponding changes in the code?

Reply:
It appears so, which I'm guessing is why we have the failure. I'm trying to figure out where the changes should be made.

Reply:
I'm just curious why these IDs would change? What's the significance of migrating 0 to 1, 1 to 2, 2 to 3, 3 to 4, 4 to 5, and 5 to 23? For what reason would we do this?

Reply:
BTW not that on a few of the skeleton templates there are tests that could break

Reply:
@cquiroz Thanks. I have been trying to fix the tests as well, but reached a point on one of thee two (3660) where the algorithm just seems wrong.